### ISSUE TITLE
Does gophercloud utils support gnocchi aggregate?

### ISSUE THREAD

**Issue by FakeKuryr** [gophercloud/utils#163]
I find how to get instance cpu_util (Instance percentage of host cpu ) from stackoverflow https://stackoverflow.com/a/57386023/12524552
But I cant't find about aggregate interface in gophercloud/utils/gnocchi/metric/v1/
So utils doesn't support aggregate like this ["(* (aggregate mean (metric cpu.util mean)) 4)"]?(https://gnocchi.osci.io/rest.html#examples)

**Comment by jtopjian** [comment]
Yeah, it looks like we're missing the `/v1/aggregates` APIs. I'd be happy to review a PR for anyone who wants to submit it.

**Comment by pierreprinetti** [comment]
Please [open an issue in the Gophercloud repository](https://github.com/gophercloud/gophercloud/issues/new/choose) in case it's still relevant
